examples/Carbon_Flux.ipynb

```diff
@@ -158,7 +200,7 @@
    "outputs": [],
    "source": [
     "import scipy.io\n",
-    "rsif = scipy.io.loadmat('RSIF_2007_2016_05N_01L.mat')"
+    "rsif = scipy.io.loadmat('./flux_data/RSIF_2007_2016_05N_01L.mat')"
```

Comment:
I note you've changed where to expect this `.mat` file. Either option is fine though this way means overwriting the git lfs 'file' of the same name...